.github/workflows/ci-dockers.yml

```diff
@@ -42,9 +42,10 @@ jobs:
         include:
           # We only release one docker image per PyTorch version.
           # The matrix here is the same as the one in release-docker.yml.
```

Comment:
According to this comment, if you touch these configs, you should update `release-docker.yml` too

Reply:
updated